Scripts/lem.py

```diff
@@ -171,28 +171,29 @@ def main(args):
         "--version",
         action="version",
         version="helmet " + str(config.VERSION))
+    parser.add_argument(
+        "--json",
+        type=str,
+        help="Read parameters from file, override command-line and dev-config.json arguments",
+    )
     # Logging
     parser.add_argument(
         "--log-level",
         choices={"DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"},
-        default=config.LOG_LEVEL,
```

Comment:
We could save all possible parameters in a list of NamedTuples. Then we could just iterate that list here and set the parser arguments:
`for a in config.arguments: parser.add_argument(a.cmd_flag, type=a.type, default=a.default, help=a.help)`
and use the same list in other places like the config.py